playwright/_impl/_helper.py

```diff
@@ -275,3 +275,21 @@ def to_impl(obj: T) -> T:
     if hasattr(obj, "_impl_obj"):
         return cast(Any, obj)._impl_obj
     return obj
+
+
+def object_to_array(obj: Optional[Dict]) -> Optional[List[NameValue]]:
+    if not obj:
+        return None
+    result = []
+    for key, value in obj.items():
+        result.append(NameValue(name=key, value=str(value)))
```

Comment:
I'm sure we have such method for headers (same types)

Reply:
headers have more edge-cases for e.g. set-cookies etc.